plugins/brew/brew.plugin.zsh

```diff
@@ -1,11 +1,19 @@
+if (( $+commands[brew] )); then
+  return
+fi
+
+if [[ -z "$HOMEBREW_PREFIX" ]]; then
+  eval "$(brew shellenv)"
```

Comment:
I think we should support running brew by checking the absolute path to the executable, if the command is not found. 

That means rewriting the initial check and only bailing out if neither command is found (`/opt/homebrew/bin/brew` or `/usr/local/bin/brew`).

I just realised that Homebrew can also be installed in Linux, maybe we should also check for that (see linuxbrew).

Reply:
Hmm looks nice. I'll update it tomorrow 👌🏻